packages/webservice/app/src/test/kotlin/com/yahoo/navi/ws/test/integration/StaticAssetServingTest.kt

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+package com.yahoo.navi.ws.test.integration
+
+import com.jayway.restassured.RestAssured
+import com.jayway.restassured.RestAssured.given
+import com.yahoo.navi.ws.test.framework.IntegrationTest
+import org.hamcrest.CoreMatchers.equalTo
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+class StaticAssetServingTest : IntegrationTest() {
+
+    @BeforeEach
+    override fun setUp() {
+        super.setUp()
+        RestAssured.basePath = "/"
+    }
+
+    @Test
+    fun landing_page_test() {
+        val content = this.javaClass::class.java.getResource("/META-INF/resources/index.html").readText()
+
+        given()
+            .header("User", "testuser")
+            .When()
+            .get("/")
+            .then()
+            .log().all()
+            .body(equalTo(content))
+            .statusCode(200)
```

Comment:
Can we assert the redirect url?

Reply:
I tested that this was working manually with a local server.  I wasn't able to capture the redirect with Rest Assured.